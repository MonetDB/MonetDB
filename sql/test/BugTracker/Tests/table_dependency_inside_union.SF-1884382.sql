create table t1884382a (id int);
create table t1884382b (id int);

create view v1 as (select * from t1884382a
                        union
                   select * from t1884382b);

drop table t1884382a;
drop table t1884382b;

drop view v1;

select * from t1884382a;
drop table t1884382a;
drop table t1884382b;
