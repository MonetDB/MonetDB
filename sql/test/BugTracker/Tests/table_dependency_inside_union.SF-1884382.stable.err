stderr of test 'table_dependency_inside_union.SF-1884382` in directory 'sql/test/BugTracker` itself:


# 14:42:28 >  
# 14:42:28 >   mserver5 "--config=/ufs/goncalve/scratch/MonetDB/debug/MonetDB5/etc/monetdb5.conf" --debug=10 --set "monet_mod_path=/ufs/goncalve/scratch/MonetDB/debug/MonetDB5/lib/MonetDB5:/ufs/goncalve/scratch/MonetDB/debug/MonetDB5/lib/MonetDB5/lib:/ufs/goncalve/scratch/MonetDB/debug/MonetDB5/lib/MonetDB5/bin" --set "gdk_dbfarm=/ufs/goncalve/scratch/MonetDB/debug/MonetDB5/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/goncalve/scratch/MonetDB/debug/MonetDB5/var/MonetDB5/sql_logs"  --set mapi_open=true --set xrpc_open=true --set mapi_port=32819 --set xrpc_port=42261 --set monet_prompt= --trace  "--dbname=mTests_src_test_BugTracker" --set mal_listing=0 "--dbinit= include sql;" ; echo ; echo Over..
# 14:42:28 >  

#warning: please don't forget to set your vault key!
#(see /ufs/goncalve/scratch/MonetDB/debug/MonetDB5/etc/monetdb5.conf)

# 14:42:29 >  
# 14:42:29 >  mclient -lsql -umonetdb -Pmonetdb --host=amelia --port=32819 
# 14:42:29 >  

MAPI  = (monetdb) /var/tmp/mtest-27483/.s.monetdb.35395
QUERY = drop table t1884382a;
ERROR = !DROP TABLE: unable to drop table t1884382a (there are database objects which depend on it)
CODE  = 42000
MAPI  = (monetdb) /var/tmp/mtest-30274/.s.monetdb.37685
QUERY = drop table t1884382b;
ERROR = !DROP TABLE: unable to drop table t1884382b (there are database objects which depend on it)
CODE  = 42000

# 14:42:29 >  
# 14:42:29 >  Done.
# 14:42:29 >  

