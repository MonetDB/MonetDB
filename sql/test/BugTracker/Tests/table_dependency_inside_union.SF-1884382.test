statement ok
create table t1884382a (id int)

statement ok
create table t1884382b (id int)

statement ok
create view v1 as (select * from t1884382a
                        union
                   select * from t1884382b)

statement error
drop table t1884382a

statement error
drop table t1884382b

statement ok
drop view v1

query I rowsort
select * from t1884382a
----

statement ok
drop table t1884382a

statement ok
drop table t1884382b

