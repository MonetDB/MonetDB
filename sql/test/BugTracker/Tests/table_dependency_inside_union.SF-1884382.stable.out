stdout of test 'table_dependency_inside_union.SF-1884382` in directory 'sql/test/BugTracker` itself:


# 14:42:28 >  
# 14:42:28 >   mserver5 "--config=/ufs/goncalve/scratch/MonetDB/debug/MonetDB5/etc/monetdb5.conf" --debug=10 --set "monet_mod_path=/ufs/goncalve/scratch/MonetDB/debug/MonetDB5/lib/MonetDB5:/ufs/goncalve/scratch/MonetDB/debug/MonetDB5/lib/MonetDB5/lib:/ufs/goncalve/scratch/MonetDB/debug/MonetDB5/lib/MonetDB5/bin" --set "gdk_dbfarm=/ufs/goncalve/scratch/MonetDB/debug/MonetDB5/var/MonetDB5/dbfarm" --set "sql_logdir=/ufs/goncalve/scratch/MonetDB/debug/MonetDB5/var/MonetDB5/sql_logs"  --set mapi_open=true --set xrpc_open=true --set mapi_port=32819 --set xrpc_port=42261 --set monet_prompt= --trace  "--dbname=mTests_src_test_BugTracker" --set mal_listing=0 "--dbinit= include sql;" ; echo ; echo Over..
# 14:42:28 >  

# MonetDB server v5.3.0, based on kernel v1.21.0
# Serving database 'mTests_src_test_BugTracker'
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Copyright (c) 1993-2007 CWI, all rights reserved
# Visit http://monetdb.cwi.nl/ for further information
# Listening for connection requests on mapi:monetdb://amelia.ins.cwi.nl:32819/
# MonetDB/SQL module v2.21.0 loaded

#function user.main():void;
#    clients.quit();
#end main;


# 14:42:29 >  
# 14:42:29 >  mclient -lsql -umonetdb -Pmonetdb --host=amelia --port=32819 
# 14:42:29 >  

% sys.t1884382a # table_name
% id # name
% int # type
% 1 # length

# 14:42:29 >  
# 14:42:29 >  Done.
# 14:42:29 >  

